schemas/bursar-export/filter/bursarExportFilterFeeFineOwner.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Make sure you add this in to the main `bursarExportFilter`'s type mapping

Reply:
My bad. Will do!